docs/manuals/core/entities/scheduling-and-job-execution.md

```diff
@@ -160,52 +160,106 @@ This example will produce the following output:
 
 Jobs are created when a task is submitted.
 
-- You can cancel the Job with `taipy.cancel_job(job)^`. When cancelling a Job, you will effectively set the `Status^` of subsequent jobs of the cancelled Job to `ABANDONED`.
+- You can cancel the Job with `taipy.cancel_job(job)^`. When cancelling a Job, you will effectively set the `Status^` of subsequent jobs of the cancelled Job to `ABANDONED`. However, if a job is already being run, it cannot be cancelled. When cancelling a running job, its subsequent jobs will be abandoned and the running job will continue running until completed.
 
-!!! Warning
-    Cancelling a `RUNNING` Job will set the Job to `CANCELLED`. However, it will not terminate the subprocess that has already been dispatched to execute the Job.
+!!! example "Cancelling a job"
 
-!!! Example
+    ```python linenums="1"
```

Comment:
Please use the line numbering if and only if you reference a specific line or lines range.
This makes the reading better.